spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -17,3 +17,28 @@ def foobar_action
     end
   end
 end
+
+RSpec.describe Hq::PartnersController, type: :controller do
+  before :each do
+    sign_in instance_double(AdminUser)
+    @partner = build_stubbed :partner
+    expect(Partner).to receive(:find).and_return(@partner)
```

Comment:
meaning that `@partner` defined here is accessed from the controller, as we discussed in the scrum?